pkgs/by-name/pl/plasmusic-toolbar/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+  pname = "plasmusic-toolbar";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "ccatterina";
+    repo = "plasmusic-toolbar";
+    rev = "v${version}";
+    hash = "sha256-yc6hUi5tICpG3SacYnWVApYQXPN4Yrw6+BFd9ghlqxA=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/plasma/plasmoids/plasmusic-toolbar
+    cp -r $src/src/* $out/share/plasma/plasmoids/plasmusic-toolbar
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "KDE Plasma widget that shows currently playing song information and provide playback controls.";
+    homepage = "https://github.com/ccatterina/plasmusic-toolbar";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ HeitorAugustoLN ];
+    platforms = platforms.linux;
```

Perhaps it'd be a bit more correct to use:

```suggestion
   inherit (kwindowsystem.meta) platforms;
```